components/bl-treeSelect-component/src/index.js

```diff
@@ -1,33 +1,44 @@
-import { useEffect, useState } from 'react';
+import { useCallback, useEffect, useMemo, useState } from 'react';
 
 import primereact from './lib/primereact.min';
+import {
+  ensureMeasure, getSelectedItems, getSelectedKeys, updateKeysMap, updateNode, validateSelectedNodeKeys,
+} from './helpers';
 
 const { cn } = BackendlessUI.CSSUtils;
 const { TreeSelect } = primereact.treeselect;
```

Comment:
does the lib `primereact ` contain all UI components or only components we use here?

Reply:
It contains only the components we used here.

Reply:
ok, great!